modules/terraform-cdp-aws-pre-reqs/modules/vpc/variables.tf

```diff
@@ -18,6 +18,20 @@ variable "vpc_cidr" {
 
 }
 
+variable "private_cidr_range" {
```

Comment:
To expose these variables to the user, they also need to be defined in the [modules/blob/feature/cdp_provider/modules/terraform-cdp-aws-pre-reqs/variables.tf](https://github.com/jimright/terraform-cdp-modules/blob/feature/cdp_provider/modules/terraform-cdp-aws-pre-reqs/variables.tf) files.

The value specified by the user, or the default value, is then passed vpc sub-module.

Reply:
Done!